eq.yml

```diff
@@ -573,38 +615,80 @@ jobs:
         - |
           publisher_tag=$(cat eq-publisher/.git/HEAD | xargs echo -n)
           while [[ "$(curl -s https://staging-publisher.dev.eq.ons.digital/status)" != *"$publisher_tag"* ]]; do sleep 5; done
-
-- name: functional-tests-core
-  serial_groups: [staging-functional-tests-core]
+    on_failure:
+      put: slack-alert
+      params:
+        channel: '#eq-author'
+        attachments:
+          - pretext: Staging Publisher Deploy Failed
+            color: '#ff0000'
+            title: Concourse Build $BUILD_ID
+            title_link: http://concourse.dev.eq.ons.digital/builds/$BUILD_ID
+
+- name: smoke-tests
+  serial_groups: [staging-smoke-tests]
   plan:
   - aggregate:
+    - get: eq-smoke-tests
+      trigger: true
     - get: eq-survey-runner
       passed: [staging-deploy]
       trigger: true
+    - get: eq-author
+      passed: [staging-deploy]
+      trigger: true
+    - get: eq-author-api
+      passed: [staging-deploy]
+      trigger: true
+    - get: eq-publisher
+      passed: [staging-deploy]
+      trigger: true
+    - get: go-launch-a-survey
+      passed: [staging-deploy]
+      trigger: true
+    - get: eq-schema-validator
+      passed: [staging-deploy]
+      trigger: true
+    - get: eq-survey-register
+      passed: [staging-deploy]
+      trigger: true
+    - get: eq-survey-runner-deploy
+      passed: [staging-deploy]
+      trigger: true
+    - get: eq-author-deploy
+      passed: [staging-deploy]
+      trigger: true
   - task: Test
     params:
-      TRAVIS: 'true'
-      EQ_FUNCTIONAL_TEST_ENV: https://staging-new-surveys.dev.eq.ons.digital
-      EQ_FUNCTIONAL_TEST_SUITES: core
+      BASE_URL: https://staging-author.dev.eq.ons.digital
     config:
       platform: linux
       image_resource:
         type: docker-image
         source:
-          repository: onsdigital/eq-python-build
+          repository: onsdigital/eq-author-build
       inputs:
-      - name: eq-survey-runner
+      - name: eq-smoke-tests
       run:
         path: sh
         args:
         - -exc
         - |
-          cd eq-survey-runner
+          cd eq-smoke-tests
           yarn install
-          ./scripts/run_tests_functional.sh
+          yarn test_headless
+    on_failure:
+      put: slack-alert
+      params:
+        channel: '#eq-runner #eq-author'
+        attachments:
+          - pretext: Staging smoke-tests Failed
+            color: '#ff0000'
+            title: Concourse Build $BUILD_ID
+            title_link: http://concourse.dev.eq.ons.digital/builds/$BUILD_ID
 
 - name: staging-destroy
-  serial_groups: [staging-deploy, staging-functional-tests, staging-accessibility-tests, staging-performance-tests, staging-security-tests, staging-destroy]
+  serial_groups: [staging-deploy, staging-accessibility-tests, staging-performance-tests, staging-security-tests, staging-destroy]
```

Comment:
We should remove references to accessibility, performance and security tests if they don't exist yet